src/ch/ehi/umleditor/interlis/iliexport/ExportInterlis.java

```diff
@@ -44,7 +46,17 @@ public static void writeFileset()
     }
     return;
     }
-
+  
+  public static void writeIli(INTERLIS2Def ili) {
+		TransferFromUmlMetamodel writer = new TransferFromUmlMetamodel();
+		try {
+			writer.getFileList(ili.getNamespace());
+			writer.writeIliFile(ili);
+		}catch (java.io.IOException ex) {
+			ch.ehi.umleditor.application.LauncherView.getInstance().log("Export interlis file", "Fail when try write the interlis file");
```

Comment:
pls use ex.getLocalizedMessage() (so the user sees, what went wrong)

Reply:
something like this?
ch.ehi.umleditor.application.LauncherView.getInstance().log("Export interlis file", ex.getLocalizedMessage());

Reply:
yes